common.sh

```diff
@@ -174,6 +174,12 @@ fi
 
 export OPENSHIFT_RELEASE_TAG=$(echo $OPENSHIFT_RELEASE_IMAGE | sed -E 's/[[:alnum:]\/.-]*(release|okd).*://')
 
+# On 4.16+ the baremetall installer was merged into the regular installer
+export DEFAULT_OPENSHIFT_INSTALLER_CMD="openshift-install"
+if is_lower_version ${OPENSHIFT_VERSION:-""} 4.16; then
```

Comment:
OPENSHIFT_VERSION is empty in the CI run
```
 ++(common.sh:179): source(): is_lower_version '' 4.16
+++(common.sh:34): is_lower_version(): echo ' 4.16'
+++(common.sh:34): is_lower_version(): tr ' ' '\n'
+++(common.sh:34): is_lower_version(): sort -V
+++(common.sh:34): is_lower_version(): head -n1
++(common.sh:34): is_lower_version(): [[ '' != 4.16 ]]
++(common.sh:35): is_lower_version(): return 0
++(common.sh:180): source(): export DEFAULT_OPENSHIFT_INSTALLER_CMD=openshift-baremetal-install
++(common.sh:180): source(): DEFAULT_OPENSHIFT_INSTALLER_CMD=openshift-baremetal-install 
```

Reply:
Ops! It's the lowercase function `openshift_version` that returns the current payload version

Reply:
Then there's existing code that's wrong: https://github.com/openshift-metal3/dev-scripts/blob/6f6748827cb7d4f8f3d302ae99f17184cf363520/common.sh#L206